src/btree/bt_ret.c

```diff
@@ -164,8 +164,8 @@ __wt_read_row_time_pairs(
  *     Change a buffer to reference an internal original-page return value.
  */
 int
-__wt_value_return_buf(
-  WT_CURSOR_BTREE *cbt, WT_REF *ref, WT_ITEM *buf, WT_TIME_PAIR *start, WT_TIME_PAIR *stop)
+__wt_value_return_buf(WT_CURSOR_BTREE *cbt, WT_REF *ref, WT_ITEM *buf, WT_TIME_PAIR *start,
+  WT_TIME_PAIR *stop, bool *preparep)
```

Comment:
This will conflict with the changes for WT-6086 - please merge develop into that PR if this gets merged first.

Reply:
ok.